www/html/api-laravel/app/Http/Controllers/Cep/GetCepController.php

```diff
@@ -47,17 +51,27 @@ public function __construct(CepService $cepService)
     public function __invoke(Request $request)
     {
        try{
-           $cep = $request->route(self::ROUTE_CEP);
+           $cep = $request->route('cep');
```

Comment:
Topp, sugestão, como é só um parâmetro podemos fazer pegar o cep, ou da request ou de forma paramtrizada:

public function __invoke(int $cep)

```suggestion
           $cep = $cep;
```